<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1146</width>
    <height>586</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>164</green>
        <blue>68</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="NoRole">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>164</green>
        <blue>68</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="NoRole">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>164</green>
        <blue>68</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>164</green>
        <blue>68</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="NoRole">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="exportbutton">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>420</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Export!</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="FileInputBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>261</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Files</string>
    </property>
    <widget class="QListWidget" name="filelist">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>20</y>
       <width>251</width>
       <height>231</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="currentRow">
      <number>-1</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="OptionsBox">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>70</y>
      <width>261</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QRadioButton" name="radioButton_Hap">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>HAP </string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_HapQ">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>HAP  Q</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_HapAlpha">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>HAP + Alpha</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_Mjpeg">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>MJPEG</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="RenamingBox">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>70</y>
      <width>261</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Renaming Options</string>
    </property>
    <widget class="QTextEdit" name="removeInput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>231</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove Strings in Filenames</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>seperate through &quot;;&quot;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>221</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>add Prefix:</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="prefixInput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>231</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="OutputBox">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>70</y>
      <width>301</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Output</string>
    </property>
    <widget class="QTextEdit" name="Output_dir_textEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>271</width>
       <height>23</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Output Directory: </string>
     </property>
    </widget>
    <widget class="QComboBox" name="resolution_comboBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1920x1080</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1280x720</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>600x200</string>
      </property>
     </item>
    </widget>
    <widget class="QCheckBox" name="orig_resolution_checkBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Use source original resolution</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="predefined_label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Use predefined resolutions</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Use custom resolution</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>220</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>220</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="w_label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>220</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>width:</string>
     </property>
    </widget>
    <widget class="QLabel" name="w_label_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>220</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>height:</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>821</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CheesyHAP Batch Hap Converter/Exporter v 0.1 by Artur Kechter, based on ffmpeg</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>1131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="title_cheesy">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>15</y>
      <width>531</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CheesyHAP</string>
    </property>
   </widget>
   <widget class="QLabel" name="pixmap_label_7">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Cheese Place</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1146</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSIMPLE_HAP_BATCH_EXPORTER">
   <property name="text">
    <string>SIMPLE HAP BATCH EXPORTER</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>orig_resolution_checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>resolution_comboBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>930</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>930</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>orig_resolution_checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>predefined_label</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>930</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>920</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
